ephios/core/views/event.py

```diff
@@ -609,30 +602,43 @@ def form_valid(self, form):
         messages.success(self.request, _("Event copied successfully."))
```

Comment:
What we are doing to copy events (and shifts?!) is insufficient, as all of the models related to that event/shift configuration aren't copied (e.g. auto qualification, federation etc.). I think we should have some more elaborate signalling to allow plugins to copy their additional contents as well.

Reply:
We can add it later if need arises, sure.